test/lib/import/colorectal/providers/royal_marsden/royal_marsden_handler_colorectal_test.rb

```diff
@@ -0,0 +1,125 @@
+require 'test_helper'
+require 'import/genotype.rb'
+require 'import/colorectal/core/genotype_mmr.rb'
+require 'import/brca/core/provider_handler'
+require 'import/storage_manager/persister'
+
+class RoyalMarsdenHandlerColorectalTest < ActiveSupport::TestCase
+  def setup
+    @record   = build_raw_record('pseudo_id1' => 'bob')
+    @genotype = Import::Colorectal::Core::Genocolorectal.new(@record)
+    # TODO: Fully qualify CambridgeHandler in cambridge_handler.rb
```

Comment:
This appears to be referencing a different handler?

Reply:
True. A copy/paste mistake. Will push corrected version

Reply:
Thanks @FrancescoNea